<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cusp_Style_Widget</class>
 <widget class="QWidget" name="Cusp_Style_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>164</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="check_handle_length">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Curve</string>
     </property>
     <property name="buddy">
      <cstring>spin_handle_length</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="spin_handle_length">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Size of the curve control handles</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="maximum">
      <double>8192.000000000000000</double>
     </property>
     <property name="value">
      <double>24.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="check_crossing_gap"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Gap</string>
     </property>
     <property name="buddy">
      <cstring>spin_crossing_gap</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="spin_crossing_gap">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Distance between the ends of the a thread passing under another thread</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="maximum">
      <double>8192.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="check_cusp_angle"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Angle</string>
     </property>
     <property name="buddy">
      <cstring>spin_cusp_angle</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="spin_cusp_angle">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Minimum angle required to render a cusp</string>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="maximum">
      <double>359.000000000000000</double>
     </property>
     <property name="value">
      <double>225.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="check_cusp_distance"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Distance</string>
     </property>
     <property name="buddy">
      <cstring>spin_cusp_distance</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="spin_cusp_distance">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Distance of the cusp tip from the node position</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="minimum">
      <double>-8192.000000000000000</double>
     </property>
     <property name="maximum">
      <double>8192.000000000000000</double>
     </property>
     <property name="value">
      <double>32.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="check_cusp_shape"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Style</string>
     </property>
     <property name="buddy">
      <cstring>combo_cusp_shape</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="combo_cusp_shape">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Shape style</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_handle_length</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_handle_length</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_crossing_gap</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_crossing_gap</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_cusp_angle</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_cusp_angle</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_cusp_distance</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_cusp_distance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_cusp_shape</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_cusp_shape</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
